libewf/libewf_single_file_permission.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Single file source functions
+ *
+ */
+
+#if !defined( _LIBEWF_SINGLE_FILE_PERMISSION_H )
+#define _LIBEWF_SINGLE_FILE_PERMISSION_H
+
+#include <common.h>
+#include <types.h>
+
+#include "libewf_date_time.h"
+#include "libewf_libcerror.h"
+
+#if defined( __cplusplus )
+extern "C" {
+#endif
+
+typedef struct libewf_single_file_permission libewf_single_file_permission_t;
+
+struct libewf_single_file_permission
+{
+	/* The id
+	 */
+	uint32_t id;
+
+	/* The type
+	 */
+	uint8_t type;
+
+	/* The name
+	 */
+	uint8_t *name;
+
+	/* The name size
+	 */
+	size_t name_size;
+
+	/* The uid
+	 */
+	uint8_t *uid;
+
+	/* The uid size
+	 */
+	size_t uid_size;
+	
+	/* The props
+	 */
+	uint32_t props;
+
+	/* The access mask
+	 */
+	uint32_t access_mask;
```

Comment:
@jtsylve aren't these the [Windows NT Access control entry (ACE) flags](https://github.com/libyal/libfwnt/blob/master/documentation/Security%20Descriptor.asciidoc#access-control-entry-ace-flags)?

Reply:
Do you have sample data you can share?

Reply:
I was able to confirm this myself, updated the documentation.